modules/tutorials/pages/java-tutorial/first-couchbase-application.adoc

```diff
@@ -0,0 +1,191 @@
+:page-pagination:
+
+= Writing your first Couchbase application
+
+include::partial$tutorial-globals.adoc[]
+
+:crud-footnote: footnote:[Create Read Update Delete]
+
+:description: pass:q[Following on from xref:install-couchbase-java-sdk.adoc[], this tutorial will show you how to connect to Couchbase and add records to the database using the SDK.]
+
+[abstract]
+{description}
+
+== Introduction
+Working with any database SDK usually involves the following operations:
+
+. Connect to the database
+. Carry out CRUD{crud-footnote} operations
+. Disconnect from the database (important for resource cleanup)
+
+Fortunately, the Couchbase SDK follows the same pattern, so the first thing you're going to do is write the code to
+connect to your document store.
+
+== Connecting to the  database [[connecting-to-the-database]]
+
+In the xref:java-tutorial/install-couchbase-java-sdk.adoc[] tutorial, you created a directory structure to hold your application.
+Create the file `student.java` in the `java` directory as shown below.
+
+include::partial$student-mvn-directory-structure.adoc[tags="location-of-student.java"]
+
+Add the following content to the `ConnectStudent.java` file.
+
+[source, java]
+----
+import com.couchbase.client.java.Bucket;
+import com.couchbase.client.java.Cluster;
+import com.couchbase.client.java.Collection;
+import com.couchbase.client.java.Scope;
+
+import java.time.Duration;
+
+public class ConnectStudent {
+
+    public static void main(String[] args) {
+
+        Cluster cluster = Cluster.connect("localhost",
+                "Administrator", "password");    // <.>
```

Comment:
explicit numbers would be more readable for future?

Reply:
Yup